File: 05.fundamentalDatatypes/age.rb
#AGE CALCULATOR

#Time Measurements
SECONDS_IN_MINUTE = 60
MINUTES_IN_HOUR = 60
HOURS_IN_DAY = 24
DAYS_IN_YEAR = 365

#Calculate
answer = 0
print "Enter your age in years: "
age = gets.to_i
puts "You are ..."
answer = age * DAYS_IN_YEAR
print answer
puts " days old, or"
answer = answer * HOURS_IN_DAY
print answer
puts " hours old, or"
answer = answer * MINUTES_IN_HOUR
print answer
puts " minutes old, or"
answer = answer * SECONDS_IN_MINUTE
print answer
puts " seconds old."

File: 05.fundamentalDatatypes/age.cc
#include <iostream>
using namespace std;

int main()
{
    int age = 0;
    
    cout << "Enter your age in years: ";
    cin >> age; 
    
    int days = age * 365;
    int hours = age * 8760;
    unsigned long int minutes = age * 525600;
    unsigned long long int seconds = age * 31536000;
	
	cout << "You are ...\n" << days << " days old, or\n" << hours << " hours old, or\n" << minutes << " minutes old, or\n" << seconds << " seconds old.";
	return 0;
	
	
}	